cypress/integration/articleFlows/previewProfile.spec.js

```diff
@@ -0,0 +1,153 @@
+describe('Preview user profile from article page', () => {
+  describe('mobile screens', () => {
+    beforeEach(() => {
+      cy.testSetup();
+      cy.fixture('users/articleEditorV1User.json').as('user');
+
+      cy.viewport('iphone-7');
+
+      cy.get('@user').then((user) => {
+        cy.loginUser(user).then(() => {
+          cy.visit('/');
+          cy.findAllByRole('link', { name: 'Test article' })
+            .first()
+            .click({ force: true });
```

Comment:
Refactor (potentially blocking): Is the force click required here and elsewhere in the tests?

Reply:
It's not, but it's a bit of a weird case.

On an article card there are many instances of the same link, and some of them have pointer events blocked so Cypress can't click on them 🙁 

In the end, I thought it would be most robust to just take the first of the links and force the click since it's not the feature under test, but just a necessary step to get there.